Dockerfile.agent

```diff
@@ -1,4 +1,4 @@
 FROM gcr.io/distroless/static-debian11
-ARG TARGETARCH=amd64
+ARG TARGETARCH
```

Comment:
Where is this argument provided? I don't see it being provided in the Makefile at the root of the project.